templates/cases/case_read.haml

```diff
@@ -163,7 +163,10 @@
             .container-fluid
               .row
                 .contact-field-label.col-sm-6
-                  - trans "Name"
+                  - if contact_display_format == 'urn'
+                    - trans "URN"
```

Comment:
I think `get_contact_display` should always return a single value, used when a contact needs to be displayed as such. Here I think we'd include URNs in the JSON representation of a contact (provided of course that SITE_ANON_CONTACTS isn't true).